app/views/review_mailer/ask_for_review.html.erb

```diff
@@ -0,0 +1,13 @@
+<h3>Szanowna Pani/ Szanowny Panie <%= @review.person.surname %></h3>
+
+<p>W imieniu redakcji Rocznika Kognitywistycznego
+  zwracam się z prośbą o sporządzenie recenzji
+  artykułu: "<%= @review.title %>"
+  Streszczenie <%= @review.abstract %></p>
+
+<p>Akceptacja recenzji:</p>
+  <%= link_to 'Akceptuj recenzję', accepted_review_url(@review) %>
```

Comment:
Nie działa. Wyświetla taki błąd:
Missing host to link to! Please provide the :host parameter, set default_url_options[:host], or set :only_path to true